MobiFlight/Joysticks/Joystick.cs

```diff
@@ -189,9 +204,11 @@ public void Connect(IntPtr handle)
             joystick.Acquire();            
         }
 
-        virtual protected void EnumerateOutputDevices()
+        private void EnumerateOutputDevices()
         {
             Lights.Clear();
+            
+            definition?.Outputs?.ForEach(output => Lights.Add(new JoystickOutputDevice() { Label = output.Label, Name = output.Name, Byte = output.Byte, Bit = output.Bit }));
             return;
         }
```

Comment:
No longer has to happen in derived classes and can be done here based on what came from the JSON file.